Libraries/LibWeb/HTML/CanvasRenderingContext2D.cpp

```diff
@@ -910,12 +910,12 @@ void CanvasRenderingContext2D::set_filter(String filter)
                 [&](CSS::FilterOperation::DropShadow const& drop_shadow) {
                     auto resolution_context = CSS::Length::ResolutionContext::for_layout_node(*layout_node);
 
-                    float offset_x = static_cast<float>(drop_shadow.offset_x.resolved(resolution_context).raw_value());
-                    float offset_y = static_cast<float>(drop_shadow.offset_y.resolved(resolution_context).raw_value());
+                    float offset_x = static_cast<float>(drop_shadow.offset_x.resolved(resolution_context).to_px(resolution_context));
+                    float offset_y = static_cast<float>(drop_shadow.offset_y.resolved(resolution_context).to_px(resolution_context));
 
                     float radius = 0.0f;
                     if (drop_shadow.radius.has_value()) {
-                        radius = drop_shadow.radius->resolved(resolution_context).raw_value();
+                        radius = static_cast<float>(drop_shadow.radius->resolved(resolution_context).to_px(resolution_context));
```

Comment:
Is there a WPT test we can import for this? Somewhere in `html/canvas/element/filters` perhaps?

Reply:
Apparently not, but we do have an existing screenshot test so I'll update that.